<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>loginPage</class>
 <widget class="QWidget" name="loginPage">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>709</width>
    <height>534</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <property name="styleSheet">
   <string notr="true">#loginPage{
background:transparent;
}
#loginFrame {
background:rgba(30,30,30,0.6);
border-radius:15px;
}
QLineEdit {
    background: rgba(120, 120, 120, 0.5);
    border: 1px solid #ccc;
    border-radius: 15px;
    font: 11pt &quot;Noto Sans&quot;;
    padding-left: 15px;
    color: white; /* Text color */
}

QLineEdit:focus {
    border:1.5px solid #66afe9; /* Border color when focused */
    background-color: rgba(120, 120, 120, 0.7); /* Background color when focused */
}

QLineEdit::hover {
    background-color: rgba(120, 120, 120, 0.7); /* Background color on hover */
}
QPushButton {
    border-radius: 10px;
    background-color: rgba(120, 120, 120, 0.4);
    padding-left: 10px;
    padding-right: 10px;
    color: rgb(255, 255, 255);
	
	font: 11pt &quot;Noto Sans&quot;;
}

QPushButton:hover {
    background: #479EF5;
}

QPushButton:pressed {
    background-color: rgba(70, 158, 245, 0.6);
}

</string>
  </property>
  <layout class="QGridLayout" name="gridLayout">
   <item row="0" column="0">
    <widget class="QFrame" name="loginFrame">
     <property name="maximumSize">
      <size>
       <width>450</width>
       <height>450</height>
      </size>
     </property>
     <property name="frameShape">
      <enum>QFrame::Shape::StyledPanel</enum>
     </property>
     <property name="frameShadow">
      <enum>QFrame::Shadow::Raised</enum>
     </property>
     <layout class="QGridLayout" name="gridLayout_2">
      <property name="topMargin">
       <number>1</number>
      </property>
      <property name="verticalSpacing">
       <number>4</number>
      </property>
      <item row="4" column="0" colspan="2">
       <widget class="QCheckBox" name="showCheckbox">
        <property name="minimumSize">
         <size>
          <width>0</width>
          <height>30</height>
         </size>
        </property>
        <property name="maximumSize">
         <size>
          <width>16777215</width>
          <height>30</height>
         </size>
        </property>
        <property name="autoFillBackground">
         <bool>false</bool>
        </property>
        <property name="styleSheet">
         <string notr="true">QCheckBox,
QLabel,
QAbstractCheckBox,
QTreeView::indicator,
QRadioButton {
    padding: 0px;
    font-style: 12pt &quot;Noto Sans&quot;;
background:transparent;
}

QCheckBox {
    padding: 2px;
}

QGroupBox::indicator,
QMenu::indicator,
QCheckBox::indicator,
QTreeView::indicator {
    width: 15px;
    height: 15px;
    border-radius: 4px;
    border: 1px solid #606060;
    background-color: rgba(255, 255, 255, 0);
    margin-right: 5px;
}

QGroupBox::indicator:hover,
QMenu::indicator:hover,
QCheckBox::indicator:hover,
QTreeView::indicator:hover {
    background-color: rgba(255, 255, 255, 16);
}

QGroupBox::indicator:pressed,
QMenu::indicator:pressed,
QCheckBox::indicator:pressed,
QTreeView::indicator:pressed {
    background-color: rgba(255, 255, 255, 20);
    border: 1px solid #505050;
}

QGroupBox::indicator:checked,
QMenu::indicator:checked,
QCheckBox::indicator:checked,
QTreeView::indicator:checked {
    background-color: #60cdff;
    border: 1px solid #60cdff;
    image: url(:/Ressources/CheckBox.svg);
}

QGroupBox::indicator:checked:hover,
QMenu::indicator:checked:hover,
QCheckBox::indicator:checked:hover,
QTreeView::indicator:checked:hover {
    background-color: #5abce8;
    border: 1px solid #5abce8;
}

QGroupBox::indicator:checked:pressed,
QMenu::indicator:checked:pressed,
QCheckBox::indicator:checked:pressed,
QTreeView::indicator:checked:pressed {
    background-color: #53aad2;
    border: 1px solid #53aad2;
    image: url(:/Ressources/CheckBoxPressed.svg);
}

QCheckBox::indicator:indeterminate,
QGroupBox::indicator:indeterminate,
QTreeView::indicator:indeterminate,
QTableView::indicator:indeterminate {
    background-color: #60cdff;
    border: 1px solid #60cdff;
    image: url(:/Ressources/CheckBoxInterminate.svg);
}

QCheckBox::indicator:indeterminate:hover,
QGroupBox::indicator:indeterminate:hover,
QTreeView::indicator:indeterminate:hover,
QTableView::indicator:indeterminate:hover {
    background-color: #5abce8;
    border: 1px solid #5abce8;
}

QCheckBox::indicator:indeterminate:pressed,
QGroupBox::indicator:indeterminate:pressed,
QTreeView::indicator:indeterminate:pressed,
QTableView::indicator:indeterminate:pressed {
    background-color: #53aad2;
    border: 1px solid #53aad2;
    image: url(:/Ressources/CheckBoxInterminatePressed.svg);
}

QCheckBox::indicator:indeterminate:disabled,
QGroupBox::indicator:indeterminate:disabled,
QTreeView::indicator:indeterminate:disabled,
QTableView::indicator:indeterminate:disabled {
    border: 1px solid #404040;
    background-color: #404040;
    image: url(:/Ressources/CheckBoxInterminatePressed.svg);
}

QGroupBox::indicator:checked:disabled,
QMenu::indicator:checked:disabled,
QCheckBox::indicator:checked:disabled,
QTreeView::indicator:checked:disabled {
    border: 1px solid #404040;
    background-color: #404040;
    image: url(:/Ressources/CheckBoxPressed.svg); 
}

QGroupBox::indicator:disabled,
QMenu::indicator:disabled,
QCheckBox::indicator:disabled,
QTreeView::indicator:disabled {
    border: 1px solid #404040;
    background-color: rgba(255, 255, 255, 0);
}</string>
        </property>
        <property name="text">
         <string>Show password</string>
        </property>
        <property name="iconSize">
         <size>
          <width>24</width>
          <height>24</height>
         </size>
        </property>
        <property name="tristate">
         <bool>false</bool>
        </property>
       </widget>
      </item>
      <item row="2" column="0" colspan="2">
       <widget class="QLineEdit" name="username">
        <property name="minimumSize">
         <size>
          <width>0</width>
          <height>40</height>
         </size>
        </property>
        <property name="styleSheet">
         <string notr="true"/>
        </property>
        <property name="placeholderText">
         <string>Username</string>
        </property>
       </widget>
      </item>
      <item row="6" column="0" colspan="2">
       <widget class="QPushButton" name="signButton">
        <property name="minimumSize">
         <size>
          <width>0</width>
          <height>35</height>
         </size>
        </property>
        <property name="text">
         <string>Sign in</string>
        </property>
       </widget>
      </item>
      <item row="5" column="0" colspan="2">
       <widget class="QPushButton" name="loginButton">
        <property name="minimumSize">
         <size>
          <width>0</width>
          <height>35</height>
         </size>
        </property>
        <property name="styleSheet">
         <string notr="true"/>
        </property>
        <property name="text">
         <string>Login</string>
        </property>
        <property name="shortcut">
         <string>Return</string>
        </property>
       </widget>
      </item>
      <item row="3" column="0" colspan="2">
       <widget class="QLineEdit" name="password">
        <property name="minimumSize">
         <size>
          <width>0</width>
          <height>40</height>
         </size>
        </property>
        <property name="styleSheet">
         <string notr="true"/>
        </property>
        <property name="placeholderText">
         <string>Password</string>
        </property>
       </widget>
      </item>
      <item row="1" column="0" colspan="2">
       <widget class="QLabel" name="label">
        <property name="maximumSize">
         <size>
          <width>16777215</width>
          <height>50</height>
         </size>
        </property>
        <property name="styleSheet">
         <string notr="true">background:transparent;</string>
        </property>
        <property name="text">
         <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;&lt;span style=&quot; font-size:14pt; font-weight:700; color:#00aaff;&quot;&gt;Login&lt;/span&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
        </property>
        <property name="alignment">
         <set>Qt::AlignmentFlag::AlignCenter</set>
        </property>
       </widget>
      </item>
      <item row="0" column="1">
       <widget class="QToolButton" name="toolButton">
        <property name="minimumSize">
         <size>
          <width>32</width>
          <height>32</height>
         </size>
        </property>
        <property name="styleSheet">
         <string notr="true">background:transparent;
border:none;</string>
        </property>
        <property name="text">
         <string/>
        </property>
        <property name="icon">
         <iconset>
          <normaloff>:/Icons/closewhite.svg</normaloff>:/Icons/closewhite.svg</iconset>
        </property>
        <property name="iconSize">
         <size>
          <width>28</width>
          <height>28</height>
         </size>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
